src/components/MyDashboards/DashboardItem/DashboardItem.tsx

```diff
@@ -0,0 +1,26 @@
+import dashboardCommunicator from '../../../communication/dashboardCommunicator';
+import { ReactComponent as Trash } from '../../../icons/trash.svg';
+import { Dashboard } from '../../../types/entities';
+import './DashboardItem.scss';
+
+type DashbaordItemProps = Dashboard;
+
+const DashbaordItem = ({ dashboardName, dashboardId }: DashbaordItemProps) => {
+
+    const deleteDashboard = () => {
+        dashboardCommunicator.deleteDashboard(dashboardId); // update recoil
+    };
+
+    return (
+
```

Comment:
what is this whiteline

Reply:
copy paste for not writing from the beginning :)